Xamarin.Essentials/Types/DeviceCalendar.shared.cs

```diff
@@ -30,6 +30,8 @@ public class DeviceEvent
 
         public DateTimeOffset? StartDate { get; set; }
 
+        public TimeSpan Duration { get; set; }
```

Comment:
I don't think we should have three, individually set properties. Either EndDate or Duration should be a calculated value

Reply:
Dealt with in later PR